test/integration/test-amp-ad-3p.js

```diff
@@ -162,7 +161,7 @@ describe.configure().skipIfPropertiesObfuscated().run('amp-ad 3P', () => {
     }).then(() => {
       return poll('wait for attemptChangeSize', () => {
         return iframe.contentWindow.ping.resizeSuccess != undefined;
-      });
+      }, null, 5000);
```

Comment:
Hmmm I don't understand. If we don't want to fail the test on this assertion, it's same as we mute it. What's is the difference?

Reply:
this line change is actually not relevant to the above assertion that I unmute. just found this `poll` is flaky out on Travis.

Reply:
Updated PR description to better describe the intention of the PR.